pkg/templates/templates.go

```diff
@@ -84,6 +88,7 @@ type Config struct {
 	LookupNamespace     string
 	StartDelim          string
 	StopDelim           string
+	InputIsYaml         bool
```

Comment:
I believe that this would better follow the Go naming convention:
```suggestion
	InputIsYAML         bool
```